js/helpers.js

```diff
@@ -371,14 +371,6 @@ Helpers.UpdateHtmlFontAttributes = function ($epubHtml, fontSize, fontObj, callb
                     "rel" : "stylesheet",
                     "type" : "text/css"
                 });
-                if (!('onload' in link[0])) {
-//alert("LINK ONLOAD !?");
-                    var imgTag = $epubHtml[0].ownerDocument.createElement("img");
-                    imgTag.onerror = fontLoadCallback_;
-                    imgTag.src = fontObj.url;
-                } else {
-                    link[0].onload = fontLoadCallback_;
-                }
```

Comment:
Yes, I'm aware that when there's some latency, there might be a flash when the font is finally loaded. I wouldn't worry too much about the fontLoadCallback_ being called prematurely, as it doesn't depend on font family CSS information. And as you point out, any change will be captured. 
I can try and restore "onload" listeners on font stylesheets, but there was a flicker even with these in place. Maybe a better load listener would be on the font file instead of the font stylesheet. What do you think?